judges/find-missing-open-issues/find-missing-open-issues.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 Zerocracy
+# SPDX-License-Identifier: MIT
+
+# Judge that monitors facts with exists repository and issue properties and
+# create missing [issue/pull]-was-opened fact
+
+require 'fbe/octo'
+require 'fbe/conclude'
+require 'fbe/issue'
+require 'fbe/who'
+
+%w[issue pull].each do |type|
+  what = "#{type}-was-opened"
+  Fbe.conclude do
+    quota_aware
+    on "(and
+          (eq where 'github')
+          (exists repository)
+          (exists issue)
+          (not (eq what '#{what}'))
+          (matches what '#{type}')
```

Comment:
@Yegorov this predicate will make this query extremely slow. Imagine, 10K records and all of them will need a regular expression match. Indexes won't be used.

Reply:
@yegor256 ok, but another solution is to add this check to the `consider` block. Or maybe it can be done in some other way?